### PR TITLE
Update redis firefighting

### PR DESCRIPTION
**Pull request by dannyroberts**
https://dimagi-dev.atlassian.net/browse/SAAS-11421

##### ENVIRONMENTS AFFECTED
none

### CODE REVIEW

**Review by biyeun — APPROVED** [review]

File: docs/firefighting/index.md [hunk 1]
```diff
@@ -809,6 +811,36 @@ OK
...
+127.0.0.1:6379> ping
+PONG
+```
+once it responds with `PONG` redis is back up and ready to serve requests.
+
+## Tail the redis log
+
+To show the last few lines of the redis log during firefighting you can run:
+```
+cchq <env> run-shell-command redis 'tail /opt/data/redis/redis.log'
```
Comment (knguyen142) [inline docs/firefighting/index.md:841]: Might want to add the cloudwatch logs too for more/past info.

File: docs/services/redis.md [hunk 2]
```diff
@@ -9,4 +9,14 @@ for caching and locking.
...
+
+## Configuration recommendations
+
+### Disk
+
+You should allocate at least 3x as much disk for redis data as the redis `maxmemory` setting.
+
+If you define a `redis_maxmemory` variable in your environment's public.yml then that will be the value. Otherwise it will be half of the total memory of the machine.
+
+So for example if `redis_maxmemory` is not set and you're running redis on an 8G machine, then redis's `maxmemory` setting will be 4G and you should allocate at least 4G x 3 = 12G of disk for redis data (in addition to whatever other disk space needed for the OS, logs, etc.). This will allow enough room for redis's AOF (data persistence file), whose rewrite process makes it oscillate between at most `maxmemory` and at most `3 x maxmemory` in a saw-tooth fashion.
```
Comment (knguyen142) [inline docs/services/redis.md:22]: Can you use `GB` instead of `G`?

**Review by knguyen142 — APPROVED** [review]
Was skimming through this document because I had never seen it, and there's a reference to anisible vault on line 38 that we might as well update.

File: docs/firefighting/index.md (hunk 1 above)
Comment (dannyroberts) [reply docs/firefighting/index.md:841]: Okay yeah good point, will add

File: docs/services/redis.md (hunk 2 above)
Comment (dannyroberts) [reply docs/services/redis.md:22]: Yeah sure

File: docs/firefighting/index.md (hunk 1 above)
Comment (dannyroberts) [reply docs/firefighting/index.md:841]: https://github.com/dimagi/commcare-cloud/pull/4314

**Pull request merged**